lab/changelog.md

```diff
@@ -4,6 +4,56 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+## Lab v2.5.0 - <TODO: Add the release date>
+
+## What's New
+
+* In case of multiple Cypher queries in the query editor, you will be able to select a single query and run
+  just that one without a need to comment out all the other Cypher queries.
+
+<TODO: Add screenshot here>
+
+* A limit of having one query execution view has been removed and you can have multiple query executions now. It makes
+  life easier especially when you want to compare different Cypher queries side to side in terms of speed or
+  query results.
+
+<TODO: Add screenshot here>
+
+* Showing only the last result when running multiple Cypher queries at once has been changed. Now, results for each
+  Cypher query will be returned and you will be able to select and view each one.
+
+<TODO: Add screenshot here>
+
+* When using code blocks in rich collection descriptions, you can specify one of the following languages to have
+  syntax highlighting enabled: `cypher`, `bash`, `python`, `css`, `c`, `cpp`, `json`, `sql`, and `yaml`. Check the
+  examples for syntax highlighting on [Markdown Guide](https://www.markdownguide.org/extended-syntax/#syntax-highlighting).
+
+<TODO: Add screenshot here>
+
+* Query modules are now sorted alphabetically which will enable easier and faster. A search has been added for query
+  module procedures if there are more than 5 procedures within a module for easier narrowing down (e.g. `nxalg`
+  with [49 procedures](https://memgraph.com/docs/mage/query-modules/python/nxalg)).
+* Rendering graph view for medium/larger graphs might take time, so for any graph with more than 5k nodes or 10k edges
+  you will be asked if you want to proceed with the graph visualization or switch to the data view. This also fixed
+  showing a default graph view for each new query execution and accidentally rendering the full database graph.
```

Comment:
```suggestion
* When rendering a graph with more than 5k nodes or 10k relationships, which might take considerable amount of time to preview, you will be asked if you want to proceed with the graph visualization or switch to the data view. Due to this feature, graph view is no longer a default result view. 
```

Reply:
I'm not actually sure what this means: This also fixed
  showing a default graph view for each new query execution and accidentally rendering the full database graph.

Reply:
I will take your suggestion, but I will remove the last sentence "Due to this feature, graph view is no longer a default result view" because it is not entirely true.

To sync with you on that so you are aware of how it works:

* When you open query execution and run a query such as `MATCH (n)-[e]->(m) RETURN n, e, m` - this query returns nodes and relationships which means "Graph view" will be shown and it will start rendering the graph (the graph might be 1M nodes and 1M relationships)
* When you open query execution and run a query such as `MATCH (n)-[e]-(m) RETURN n.name` - this query returns a primitive, name. The default view will be "Data view" because Lab can't show a graph view as there are no nodes and relationships in the result.
* Within this query execution, if the user switches between Data and Graph view, that view will be the one that is favored. E.g. user switches to Data view and runs another query that returns nodes and relationships (Graph view can render it), the Lab will still show the last user selection, which was Data view.

The feature above will just add a protection layer for rendering large graphs and waiting for it to complete the render. So whenever graph view is selected and there is a bunch of nodes and relationships there will be an action: "Are you sure you want to proceed". 
Just having this will also fix issues that users had where they would have millions of nodes/edges and they would open new query execution and run `MATCH (n)-[e]->(m) RETURN n, e, m` and then rendering starts which takes a lot of time for millions of nodes/relationships.

Reply:
I also agree you can just leave that last sentence out! :)